plugins/formal-verification/testData/diagnostics/no_contracts/classes/unique_predicates.kt

```diff
@@ -2,11 +2,26 @@
 // NEVER_VALIDATE
 
 import org.jetbrains.kotlin.formver.plugin.Unique
+import org.jetbrains.kotlin.formver.plugin.Borrowed
 
 class T()
 
 open class S()
 
 class Foo(val w: Int, var x: Int, @property:Unique val y: T, @property:Unique var z: T) : S()
 
-fun <!VIPER_TEXT!>use_unique_foo<!>(@Unique foo: Foo) {}
+fun <!VIPER_TEXT!>unique_foo_arg<!>(@Unique foo: Foo) {}
+
+fun <!VIPER_TEXT!>nullable_unique_arg<!>(@Unique t: T?) {}
+
+fun <!VIPER_TEXT!>borrowed_unique_arg<!>(@Unique @Borrowed t: T) {}
+
+fun @receiver:Unique T.<!VIPER_TEXT!>unique_receiver<!>() {}
+
+fun @receiver:Unique @receiver:Borrowed T.<!VIPER_TEXT!>borrowed_unique_receiver<!>() {}
```

Comment:
@GrigoriiSolnyshkin Are receivers sufficiently supported for this test to make sense?

Reply:
In current version of our `formal-verification` branch extension functions don't work because there's a couple of places in the code where we only consider dispatch receivers. With one of my open PRs the issue seems to be solved not only for inline extensions but also for non-inline ones (probably should add some testcases for that then). However, functions with two receivers are not supported yet. 
Do you actually need something special from the receivers specifically comparing to simple arguments? Or are these test mostly added to check the annotations?

Reply:
> Do you actually need something special from the receivers specifically comparing to simple arguments? Or are these test mostly added to check the annotations?
Nothing special comparing to a simple argument, I just added them to test the annotations. For the moment they are not even called.

Reply:
I guess it's fine to leave them then